Bukkit/src/main/java/net/minecrell/serverlistplus/bukkit/BukkitPlugin.java

```diff
@@ -92,6 +95,8 @@
     private MetricsLite metrics;
 
     private Method legacy_getOnlinePlayers;
+    
+    @Getter @Setter private BanDetector banDetector = new BukkitBanDetector();
```

Comment:
Can we make this part of `ServerListPlusCore`? If this getter/setter is available for all platforms, there is no need to add this separately for each platform.

Reply:
Erm that would mean that the plugin class needs to have a method to generate the actual object. I added these methods in the interface. If you have a good idea on how to implement it let me know!